sdks/python/tests/unit/decorator/test_tracker_outputs.py

```diff
@@ -1634,3 +1634,144 @@ def f(x):
     trace_tree = fake_backend.trace_trees[0]
 
     assert_equal(EXPECTED_TRACE_TREE, trace_tree)
+
+
+def test_track__using_distributed_headers__spans_are_created_correctly(fake_backend):
+    @tracker.track
+    def inner_thread(x):
+        return "inner_thread-output"
+
+    @tracker.track
+    def top_thread(x):
```

Comment:
`top_function_in_thread` and `inner_function_in_thread` make more sense I suppose? They are both executed in the same thread, otherwise may seem like there are nested thread creations.